crates/core/src/client/mod.rs

```diff
@@ -944,29 +947,29 @@ impl KakarotClient for KakarotClientImpl {
         // Fixed fields in the Ethereum block as Starknet does not have these fields
 
         //TODO: Fetch real data
-        let gas_limit = U256::from(1_000_000);
+        let gas_limit = U256::from(GAS_LIMIT);
 
         //TODO: Fetch real data
-        let gas_used = U256::from(500_000);
+        let gas_used = U256::from(GAS_USED);
 
         //TODO: Fetch real data
-        let difficulty = U256::ZERO;
+        let difficulty = U256::from(DIFFCIULTY);
 
         //TODO: Fetch real data
-        let nonce: Option<H64> = Some(H64::zero());
+        let nonce: Option<H64> = Some(NONCE);
 
         //TODO: Fetch real data
-        let size: Option<U256> = Some(U256::from(1_000_000));
+        let size: Option<U256> = Some(U256::from(SIZE));
```

Comment:
Size should also always be None imho
I'm thinking out loud and this is kinda out of the scope of this PR:)!

But basically, some fields in the Eth block/Eth Tx/ Eth Transaction are optional and we're putting in sentinel values instead of just None, even though we have no intention of ever filling them.


TL;DR->
Some values have hardcoded values that we'll one day replace with live data,
Some values are optional and should always be None imho

wdyt

Reply:
Can you expand on why `size` should be None? according to the spec it is supposed to be part of the response body + why will its value be None if it measures the block size?

Reply:
It's supposed to be the block size is bytes,
I guess we can put is as None for now and add a todo to dynamically "size" a block -> it's simply that it's unclear how we size a KKT block for now

I guess our two options are:
- once we get a starknet block, we convert it to be an eth (kakarot) block, and then size the resulting struct, and then add the size field
- or we size directly the starknet block which is the 'reality' of the chain
- or we keep it as none for now, until we know its true use -> who wants / needs size? who uses it?